scalameta/src/main/scala/scala/meta/internal/parsers/Parsers.scala

```diff
@@ -2827,9 +2891,10 @@ private[meta] class Parser(val input: Input)(implicit val dialect: Dialect) { pa
     val stats = new ListBuffer[T]
     while (!token.is[StatSeqEnd]) {
       def isDefinedInEllipsis = { if (token.is[`(`] || token.is[`{`]) next(); statpf.isDefinedAt(token) }
-      if (token.is[Ellipsis] && ahead(isDefinedInEllipsis)) stats += ellipsis(1, statpf(token))
-      else if (statpf.isDefinedAt(token)) stats += statpf(token)
-      else if (!token.is[StatSep]) syntaxError(errorMsg, at = token)
+      if (statpf.isDefinedAt(token) || (token.is[Ellipsis] && ahead(isDefinedInEllipsis)))
+        stats += statpf(token)
+      else if (!token.is[StatSep])
+        syntaxError(errorMsg, at = token)
```

Comment:
Hmm I'm having hard time seeing how `ellipsis(1, statpf(token))` and `statpf(token)` could do the same thing :puzzled: